tests/test_rocket.py

```diff
@@ -379,6 +379,56 @@ def test_add_fins_assert_cp_cm_plus_fins(calisto, dimensionless_calisto, m):
     )
 
 
+@pytest.mark.parametrize(
+    """cdm_position, grain_cm_position, nozzle_position, coord_direction, 
+    motor_position, expected_motor_cdm, expected_motor_cpp""",
+    [
+        (0.317, 0.397, 0, "nozzle_to_combustion_chamber", -1.373, -1.056, -0.976),
+        (0, 0.08, -0.317, "nozzle_to_combustion_chamber", -1, -1, -0.92),
+        (-0.317, -0.397, 0, "combustion_chamber_to_nozzle", -1.373, -1.056, -0.976),
+        (0, -0.08, 0.317, "combustion_chamber_to_nozzle", -1, -1, -0.92),
+        (1.317, 1.397, 1, "nozzle_to_combustion_chamber", -2.373, -1.056, -0.976),
+    ],
+)
+def test_add_motor_coordinates(
+    calisto_motorless,
+    cdm_position,
+    grain_cm_position,
+    nozzle_position,
+    coord_direction,
+    motor_position,
+    expected_motor_cdm,
+    expected_motor_cpp,
+):
```

Comment:
This test lacks a docstring. Here is a suggestion:

"""
Test the method add_motor and related position properties in a Rocket instance.

This test checks the correctness of the `add_motor` method and the computed 
motor_center_of_dry_mass_position and center_of_propellant_position properties in the 
`Rocket` class using various parameters related to the motor's position, nozzle's position,
and other related coordinates. Different scenarios are tested using parameterization, 
checking scenarios moving from the nozzle to the combustion chamber and vice versa, 
and with various specific physical and geometrical characteristics of the motor.

Parameters
----------
calisto_motorless : Rocket instance
    A predefined instance of a Rocket without a motor, used as a base for testing.
cdm_position : float
    Position of the center of dry mass of the motor.
grain_cm_position : float
    Position of the grains' center of mass.
nozzle_position : float
    Position of the nozzle.
coord_direction : str
    Direction for coordinate system orientation; 
    it can be "nozzle_to_combustion_chamber" or "combustion_chamber_to_nozzle".
motor_position : float
    Position where the motor should be added to the rocket.
expected_motor_cdm : float
    Expected position of the motor's center of dry mass after addition.
expected_motor_cpp : float
    Expected position of the center of propellant after addition.

"""

Reply:
Well noted. Thanks for the suggestion, added on [c437c9e](https://github.com/RocketPy-Team/RocketPy/pull/423/commits/c437c9ebcb4d279c0f55f46ddc6aed3be3ae5b65).